docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,68 @@
+# Contributing
+
+If you would like to contribute new example apps to the `elasticsearch-labs` repo, we would love to hear from you!
+
+## Before you start
+
+Prior to opening a pull request, please:
+- Create an issue to [discuss the scope of your proposal](https://github.com/elastic/elasticsearch-labs/issues). We are happy to provide guidance to make for a pleasant contribution experience.
+- Sign the [Contributor License Agreement](https://www.elastic.co/contributor-agreement/). We are not asking you to assign copyright to us, but to give us the right to distribute your code without restriction. We ask this of all contributors in order to assure our users of the origin and continuing existence of the code. You only need to sign the CLA once.
+
+## Contributing to Python notebooks 📒
+
+### Why
```

Comment:
I went with the [5W1H](https://www.wimi-teamwork.com/blog/the-5w1h-method-project-management-defined-and-applied) method to break down the list of bulletpoints into more concise chunks.